cloud-consumption/src/test/java/com/sequenceiq/consumption/service/ConsumptionServiceTest.java

```diff
@@ -144,4 +159,149 @@ void testCreateWhenConsumptionAlreadyExists() {
         verify(consumptionDtoConverter, Mockito.times(0)).creationDtoToConsumption(eq(consumptionCreationDto));
     }
 
+    @Test
+    public void testFindAllConsumption() {
+        when(consumptionRepository.findAllConsumption()).thenReturn(List.of());
+
+        underTest.findAllConsumption();
+
+        verify(consumptionRepository, times(1)).findAllConsumption();
+    }
+
+    @Test
+    public void testNoAggregationRequiredIfNotStorageType() {
+        Consumption consumption = new Consumption();
+        consumption.setConsumptionType(ConsumptionType.UNKNOWN);
+
+        boolean result = underTest.isAggregationRequired(consumption);
+
+        assertFalse(result);
+    }
+
+    @Test
+    public void testNoAggregationRequiredIfLocationValidationFails() {
+        Consumption consumption = consumption("not_s3_location");
+        doThrow(new ValidationException("error")).when(cloudStorageLocationUtil)
+                .validateCloudStorageType(FileSystemType.S3, "not_s3_location");
+
+        boolean result = underTest.isAggregationRequired(consumption);
+
+        assertFalse(result);
+    }
+
+    @Test
+    public void testAggregationRequired() {
+        Consumption consumption = consumption("s3_location");
+        doNothing().when(cloudStorageLocationUtil).validateCloudStorageType(FileSystemType.S3, "s3_location");
```

Comment:
Like above, this is typically the default behavior for `void` return type methods.